app/enterprise/0.36-x/admin-api/vitals/vitals-prometheus-strategy.md

```diff
@@ -31,8 +32,7 @@ be downloaded at
 Then start StatsD exporter with
 
 ```bash
-$ ./statsd_exporter --statsd.mapping-config=statsd.rules.yaml \
-                    --statsd.listen-unixgram=''
```

Comment:
Will removing this option affect previous versions? Was it extraneous to begin with?

Reply:
Added back for backward compatibility 👍